pkg/sysinfo/sysinfo_linux.go

```diff
@@ -63,6 +63,27 @@ func New(quiet bool) *SysInfo {
 		}
 	}
 
+	// Check if bridge-nf-call-iptables is disabled.
+	if data, err := ioutil.ReadFile("/proc/sys/net/bridge/bridge-nf-call-iptables"); os.IsNotExist(err) {
+		sysInfo.BridgeNfCallIptablesDisabled = true
+	} else {
+		if enabled, _ := strconv.Atoi(strings.TrimSpace(string(data))); enabled == 0 {
```

Comment:
Can we please avoid the `if` here, like:

    val, _ := strconv.Atoi(strings.TrimSpace(string(data)))
    sysInfo.BridgeNfCallIptablesDisabled = val == 0